install_config/oab_broker_configuration.adoc

```diff
@@ -0,0 +1,732 @@
+
+|`auth_name`
+|Name of the secret or file storing the registry credentials that should be read.
+Used when `auth_type` is set to `secret`.
+|N footnoteref:[oabopenshiftregistrynote]
+ 
+|`user`
+|The user name for authenticating to the registry when using `auth_type: config`.
+|N
+
+|`pass`
+|The password for authenticating to the registry when using `auth_type: config`.
+|N
+
+|`org`
+|The namespace or organization that the image is contained in.
+|N
+
+|`type`
+|The type of registry. Available adapters are `mock`, `rhcc`, `openshift`,
+`dockerhub`, and `local_openshift`.
+|Y
+
+|`namespaces`
+|The list of namespaces to configure the `local_openshift` registry type with. By default, a user should use `openshift`.
+|N
+
+|`url`
+|The URL that is used to retrieve image information. Used extensively for RHCC while the `dockerhub` type uses hard-coded URLs.
+|N
+
+|`fail_on_error`
+|Should this registry fail, the bootstrap request if it fails. Will stop the execution of other registries loading.
+|N
+
+|`white_list`
+|The list of regular expressions used to define which image names should be allowed through. Must have a white list to allow APBs to be added to the catalog. The most permissive regular expression that you can use is `.*-apb$` if you would want to retrieve all APBs in a registry. See xref:oab-config-apb-filtering[APB Filtering] for more details.          
+|N
+
+|`black_list`
+|The list of regular expressions used to define which images names should never be allowed through. See xref:oab-config-apb-filtering[APB Filtering] for more details.
+|N
+
+|`images`
+|The list of images to be used with an OpenShift Container Registry.
+|N
+|===
+
+[[oab-config-registry-prod-dev]]
+=== Production or Development
+
+A _production_ broker configuration is designed to be pointed at a trusted
+container distribution registry, such as the Red Hat Container Catalog (RHCC):
+
+[source,yaml]
+----
+registry:
+  - name: rhcc
+    type: rhcc
+    url: https://registry.access.redhat.com
+    tag: v3.7
+    white_list: 
+      - ".*-apb$"
+  - type: local_openshift
+    name: localregistry
+    namespaces:
+      - openshift
+    white_list: []
+----
+
+However, a _development_ broker configuration is primarily used by developers
+working on the broker. To enable developer settings, set the registry name to
+`dev` and the `dev_broker` field in the `broker` section to `true`:
+
+[source,yaml]
+----
+registry:
+  name: dev
+----
+
+[source,yaml]
+----
+broker:
+  dev_broker: true
+----
+
+[[oab-config-registry-storing-creds]]
+=== Storing Registry Credentials
+
+The `auth_type` field in the `registry` section of the broker configuration
+determines how the broker should read the registry credentials, either the
+`config`, `secret`, or `file` type.
+
+With the `config` type, the registry credentials are read from the broker
+configuration's `user` and `pass` values in the `registry` section:
+
+----
+registry:
+  - name: openshift
```

Comment:
This name should be `isv`.

Reply:
Also should add this part: `url: registry.connect.redhat.com`

Reply:
+1 url should be prefixed with `https` too.

Reply:
:+1: